_overviews/scala3-book/scala-for-java-devs.md

```diff
@@ -964,8 +964,8 @@ Examples of how to create instances of immutable collections.
 
 ### Mutable collections classes
 
-Scala has mutable collections classes like `ArrayBuffer`, `Map`, and `Set`, in its *scala.collection.mutable* package.
-After importing them into the current scope, they’re created just like the immutable `List`, `Vector`, `Map`, and `Set` examples just shown.
+Scala has mutable collections classes like `ArrayBuffer`, `Map`, and `Set` in its _scala.collection.mutable_ package.
```

Comment:
I remember a review mentioned `Array`. I think it is very natural for Java programmers to use `Array` in Scala and should be mentioned here.

Reply:
Yes, there was another discussion somewhere about `Array`. I don’t think I got an answer to my question about where `Array` should be used, but I agree, it’s definitely natural for Java programmers to want to use it. I’ll update this text.

Reply:
Mentioning `Array` seems valid, but I would suggest that it not simply be included without additional comment. Perhaps something like: "Normal Scala code doesn't use `Array`. It is useful for interoperating with other languages (Java, JavaScript), and it may also be useful when writing low-level code that needs to squeeze maximum performance out of the underlying platform." That kind of thing.

I see a lot of beginners in our chat rooms and forums who are using `Array` not because it's actually an appropriate choice for what they're doing, but simply because they don't know any better.

Reply:
I added a discussion about `Array` and also added @SethTisue’s comments as a note after that, so I’m resolving this conversation.